app/cmd/rpc/mesh/session.go

```diff
@@ -44,19 +47,14 @@ type DispatchResponse struct {
 }
 
 // Contains - evaluate if the dispatch response contains passed address in their node list
-func (sn DispatchResponse) Contains(addr sdk.Address) bool {
-	// if nil return
-	if addr == nil {
+func (sn DispatchResponse) Contains(addr string) bool {
+	// if empty return
+	if addr == "" {
 		return false
 	}
 	// loop over the nodes
 	for _, node := range sn.Session.Nodes {
```

Comment:
converting this to a map would be more efficient in terms of CPU (with expense of slightly more memory) if you store it in the dispatch response.

though it is a premature optimization given it's only 24 nodes